application/common/components/Emailer.php

```diff
@@ -30,7 +30,7 @@ class Emailer extends Component
     const SUBJECT_MFA_OPTION_REMOVED_DEFAULT = 'A 2-Step Verification option was removed from your %s account';
     const SUBJECT_MFA_ENABLED_DEFAULT = '2-Step Verification was enabled on your %s account';
     const SUBJECT_MFA_DISABLED_DEFAULT = '2-Step Verification was disabled on your %s account';
-
+    const SUBJECT_METHOD_VERIFY_DEFAULT = 'Verification required - New account recovery method added';
```

Comment:
If possible, it may be worth rewording this subject to be less computer-ese and more normal-person-English. Perhaps something like "Please verify your new recovery method", but feel free to further tweak that.

Reply:
Good call. I'll do that.